src/app.js

```diff
@@ -37,6 +37,8 @@ const learnerServiceLocalBaseUrl = process.env.sunbird_learner_service_local_bas
 
 const whiteListedChannelList = process.env.sunbird_content_service_whitelisted_channels
```

Comment:
now that metafilters are added, channel filters can be removed right?

Reply:
Will be removed